addon/components/time-series-chart.js

```diff
@@ -163,8 +164,8 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
   // Since selected interval and time delta don't use the same naming convention
   // this converts the selected interval to the time delta convention for the
   // padding functions.
-  _getTimeDeltaFromSelectedInterval: function() {
-    switch (this.get('selectedInterval')) {
+  _getTimeDeltaFromSelectedInterval: function(interval) {
```

Comment:
This function is no longer strictly dependent on the value of `selectedInterval`, maybe rename to `_getTimeDeltaFromInterval`.